### PR TITLE
[W5][W13-2]Yang Yiqing

### PR DESCRIPTION
**Pull request by ivyyangyq**
Add a random function to randomly display a person from the list
Created new Class RandomCommand.
Updated HelpCommand, ParserCommand to include RandomCommand.
Updated UserGuide.
Updated i/o test (input.txt, expected.txt)
Updated JUnit test

Ready for Review

### CODE REVIEW

**Comment by joel0111** [comment]
Could consider splitting the commits into smaller commits :)

File: src/seedu/addressbook/commands/RandomCommand.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+     * @return list of persons found
+     */
+    private List<ReadOnlyPerson> getRandomPerson() {
+        List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+        final List<ReadOnlyPerson> Persons = new ArrayList<>();
+        if (allPersons.size() == 0) {
+            return allPersons;
+        }
+        Persons.add(allPersons.get(new Random().nextInt(allPersons.size())));
```
Comment (joel0111) [inline src/seedu/addressbook/commands/RandomCommand.java:38]: Could consider splitting up this code into 3 steps, to improve readability :)
Persons.add(allPersons.get(new Random().nextInt(allPersons.size())));
To:
int randomPersonIndex = Random().nextInt(allPersons.size());
Person randomPerson = allPersons.get(randomPersonIndex);
Persons.add(randomPerson);

File: test/java/seedu/addressbook/commands/RandomCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import java.util.Random;
+import java.util.Collections;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+
+public class RandomCommandTest {
```
Comment (tristonpang) [inline test/java/seedu/addressbook/commands/RandomCommandTest.java:16]: Good effort at writing JUnit test.

**Review by tristonpang — COMMENTED** [review]
Overall good effort @ivyyangyq ! Good attempt at implementing JUnit tests for your new command. Please close this PR when you've seen all comments.

**Pull request closed without merging**